frontend/src/pages/Player/Toolbar/DevToolsWindowV2/ConsolePage/ConsolePage.tsx

```diff
@@ -115,139 +111,21 @@ export const ConsolePage = ({
 		analytics.track('session_view-console-logs')
 	}, [])
 
-	const virtuoso = useRef<VirtuosoHandle>(null)
-
-	// eslint-disable-next-line react-hooks/exhaustive-deps
-	const scrollFunction = useCallback(
-		_.debounce((index: number) => {
-			requestAnimationFrame(() => {
-				if (virtuoso.current) {
-					virtuoso.current.scrollToIndex({
-						index,
-						align: 'center',
-					})
-				}
-			})
-		}, THROTTLED_UPDATE_MS),
-		[],
-	)
-
-	useEffect(() => {
-		if (
-			isPlayerReady && // ensure Virtuoso component is actually rendered
-			virtuoso.current &&
-			messagesToRender
-		) {
-			if (autoScroll) {
-				if (lastActiveLogIndex >= 0) {
-					scrollFunction(lastActiveLogIndex)
-				}
-			}
-		}
-	}, [
-		lastActiveLogIndex,
-		isPlayerReady,
-		messagesToRender,
-		scrollFunction,
-		autoScroll,
-	])
-
 	return (
 		<Box height="full">
-			{loading || !isPlayerReady ? (
-				<LoadingBox />
-			) : messagesToRender?.length ? (
-				<Virtuoso
-					ref={virtuoso}
-					overscan={1024}
-					increaseViewportBy={1024}
-					data={messagesToRender}
-					className={styledVerticalScrollbar}
-					itemContent={(_index, logEdge: SessionLogEdge) => (
-						<MessageRow
-							key={logEdge.cursor}
-							logEdge={logEdge}
-							current={_index === lastActiveLogIndex}
-							past={_index <= lastActiveLogIndex}
-							onSelect={() => {
-								setSelectedCursor(logEdge.cursor)
-								const timestamp =
-									new Date(logEdge.node.timestamp).getTime() -
-									sessionMetadata.startTime
-								setTime(timestamp)
-								analytics.track('session_go-to-log_click')
-							}}
-						/>
-					)}
-				/>
-			) : (
-				<Box p="8" height="full">
-					<EmptyDevToolsCallout kind={Tab.Console} filter={filter} />
-				</Box>
-			)}
+			<ConsoleTable
```

Comment:
just did a click test and this does seem quite laggy when resizing, but i'm also concerned about the performance when playing with autoscrolling. will do some more performance profiling rn - let's wait to merge this

Reply:
![Screenshot from 2024-09-24 10-37-03](https://github.com/user-attachments/assets/4e6ed249-fe9c-42e6-82e2-b4c3cf2a51dc)

looks like just playing a session, even with the dev tools minimized and auto scroll turned off, we aren't able to keep up with the time state updates (which happen every 100ms). we'll need to make sure the new logs page can render faster

Reply:
![Screenshot from 2024-09-24 10-53-59](https://github.com/user-attachments/assets/5bdf5b7f-4994-4ede-9a56-aef4bb9c21d9)
looks like the console page rendering is roughly 30% of all player rendering time

Reply:
ok... just tested on main and it seems like we have the same not great performance, so i think it's safe to say that the issue isn't as a result of this change
![Screenshot from 2024-09-24 10-59-59](https://github.com/user-attachments/assets/ee36b26e-b50b-4628-aa2f-2e8af4a18509)